addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -247,6 +346,21 @@ odoo.define('website_slides.quiz', function (require) {
                 this.trigger_up('slide_go_next');
             }
         },
+        /**
+         * Triggering a event to reset the completion of the slide
```

Comment:
I would go for something more explicit like:
"Resets the completion of the slide so the user can take the quiz again"